src/Data/VerifiedEq/Instances/Generics.hs

```diff
@@ -93,18 +99,21 @@ eqPar1Trans eqP eqPTrans x y z
   ==. eqPar1 eqP x z
   *** QED
 
--- {-@ veqPar1 :: VerifiedEq p -> VerifiedEq (Par1 p) @-}
 veqPar1 :: VerifiedEq p -> VerifiedEq (Par1 p)
 veqPar1 (VerifiedEq eqP eqPRefl eqPSym eqPTrans)
   = VerifiedEq (eqPar1      eqP)
                (eqPar1Refl  eqP eqPRefl)
                (eqPar1Sym   eqP eqPSym)
                (eqPar1Trans eqP eqPTrans)
 
+
+
+
+
```

Comment:
Are these newlines necessary?

Reply:
No, sorry -- is there any easy way to remove?

Reply:
Good to know. I'll do this myself after merging.